library/globals.inc.php

```diff
@@ -2372,7 +2378,7 @@
 
     'SMTP_PASS' => array(
       xl('SMTP Password for Authentication'),
-      'pass',                           // data type
+      'text',                           // data type
```

Comment:
You are clobbering my last commit

Reply:
I will rebase and have another go...I missed this.